pytest_splunk_addon/plugin.py

```diff
@@ -223,3 +226,9 @@ def pytest_exception_interact(node, call, report):
     if call.excinfo.type in node.session.__exc_limits:
         # pytest exits only for exceptions defined in EXC_MAP
         pytest.exit(f"Exiting pytest due to: {call.excinfo.type}")
+
+
+def pytest_sessionfinish(session, exitstatus):
```

Comment:
You can reuse this https://github.com/splunk/pytest-splunk-addon/blob/develop/pytest_splunk_addon/splunk.py#L1009 instead of adding another hook

Reply:
done.